src/mono/sample/wasm/browser-advanced/main.js

```diff
@@ -31,7 +31,7 @@ try {
         // here we show how emscripten could be further configured
         // It is preferred to use specific 'with***' methods instead in all other cases.
         .withModuleConfig({
-            configSrc: "./_framework/blazor.boot.json",
+            configSrc: "./blazor.boot.json",
```

Comment:
Why is the _framework prefix gone now?

Reply:
Because our script now lives in the `_framework` folder

Reply:
OK, so this path is relative to the script, not the application?

Reply:
Yes, this is relative to the `dotnet.js` script.
By "Application" you mean `<base>` ?

Reply:
That's idiomatic for JavaScript. All the C# is running "inside" `dotnet.js` and so all imports are relative to that. Now that we are moving the `dotnet.js` into nested folder could be surprising.

Reply:
This sort of "paths relative to the js file" resolution logic is AFAIK new to modules. I've never seen it before reviewing this PR.